### ISSUE TITLE
Separate Contribution Level From Administration Level

### ISSUE THREAD

**Issue by mcrosson** [opencollective/opencollective#1801]
## User story 
Core Contributor should be marked as such on the collective but should not have Administration rights as it's outside the contributors role in the collective. This also helps with minimizing possible "going rogue" situations if a member of a collective decides they want to harm their collective in any way.

## Best solution for this problem
Add a 2nd drop down for "access level" or a way to assign different roles (like copy editing, tier management, expense management).

**Comment by piamancini** [comment]
Yes, we currently have an issue with WWCode that are adding the collective director's as contributors (without admin rights) and they are not showing up in the contributor list. 

cc/ @xdamman

**Comment by xdamman** [comment]
I agree it would be nice to have.
Not a simple project though. It will touch on a lot of things in the code.

**Comment by alanna** [comment]
This should be fixed based on the decision we made to eliminate 'contributors' as a category of team member, and instead have only 'core contributors' and a seperate setting to designate which of them are also admins.

**Comment by Betree** [comment]
This has been resolved for some time: nowadays, core contributors don't get any additional rights, they're just surfaced as part of the team on public profiles. An additional community manager role has been introduced to surface these people who should be allowed to post updates for the collective.
